tests/acceptance/dummy-test.js

```diff
@@ -1,53 +1,49 @@
-import { test } from 'qunit';
-import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { currentURL, visit } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { percySnapshot } from 'ember-percy';
 
-moduleForAcceptance('Acceptance | dummy');
+module('Acceptance | dummy', function(hooks) {
+  setupApplicationTest(hooks);
 
-test('visiting /', function(assert) {
-  visit('/');
-  andThen(function() {
+  test('visiting /', async function(assert) {
+    await visit('/');
     assert.equal(currentURL(), '/');
+    percySnapshot('dummy homepage test');
   });
-  percySnapshot('dummy homepage test');
-});
 
-test('duplicate snapshots are skipped', function(assert) {
-  visit('/');
-  andThen(function() {
+  test('duplicate snapshots are skipped', async function(assert) {
+    await visit('/');
     assert.equal(currentURL(), '/');
+    percySnapshot('dupe test');
+    // Test duplicate name (should log warning and skip this snapshot):
+    percySnapshot('dupe test');
```

Comment:
How do we know the snapshot is skipped instead of overwritten? Maybe it doesn't matter? I guess I always assumed it was just overwritten... 
We also are not testing the log output so not sure if this test is useful?

Reply:
Per discussion, this is a regression test for making sure that dupe tests are handled correctly. Yeah it could be more thoroughly tested via log output, but there are many places in ember-percy where we integration test things anyway.